app/models/cautionary_alert.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CautionaryAlert
+  include ActiveModel::Model
+
+  attr_accessor :propertyReference, :alerts
+
+  class << self
+    def for_property_reference(reference)
+      response = PlatformApis::CautionaryAlerts::Client.
+        get_cautionary_alerts_by_property_reference(reference)
+
+      build(response)
+    end
+
+    def build(attributes)
```

Comment:
will need to check if alerts for this property exist 
and if there are no alerts, then just pass an empty object, otherwise front end is just blowing up when doing a request

```
    def build(attributes)
      if attributes["alerts"].empty?
        return new(
          propertyReference: attributes["propertyReference"],
          alerts: {}
        )
      end
      new(
        propertyReference: attributes["propertyReference"],
        alerts: Alert.build(attributes["alerts"].first)
      )
    end
  end
```